src/SonarScanner.MSBuild.Tasks/Targets/SonarQube.Integration.targets

```diff
@@ -318,10 +341,12 @@
        Configuration="$(Configuration)"
        Platform="$(Platform)"
        TargetFramework="$(TargetFramework)" />
+
+    <Message Importance="high" Text="Sonar: ($(MSBuildProjectFile)) Project processed successfully" />
```

Comment:
This is the only new message at importance `high` i.e. it will appear even in minimal log output. It seemed reasonable to have something in the least-verbose log to indicate success, since we can't easily add a message for the "not ok" case.